how-to/office-365-documentation/create-documentation.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Create Documentation
+description: >-
+  This article describes how to produce Office 365 settings documentation using
+  SysKit Trace.
+author: Vinko Bedek
+date: 01/17/2020
+---
+
+# Create Tenant Documentation
+
+This section describes how you can generate Office 365 tenant settings documentation using SysKit Trace.
+
+1. Navigate to the **Home Screen** and click the **View Snapshots** button to view all of your snapshots. Double click the snapshot to open it and navigate to the [Office 365 Explorer](../../get-to-know-syskit-trace/office-365-explorer.md). If you navigate to the Office 365 Explorer without opening a snapshot the newest snapshot will be opened automatically.
+2. Click **Generate** button in the ribbon on the top part of the screen. You can select the format in which you wish to create your tenant documentation by clicking on the dropdown. The available formats are **.docx, .pdf and .xlsx**.
```

Comment:
Maybe note what is available in trial and what is not?

Reply:
actually, as far as the documentation file goes, everything is available in the trial. single report xlsx export and word export is not supported in the trial.